api/controllers/PostController.js

```diff
@@ -348,6 +348,34 @@ module.exports = {
       )
     }))
     .then(() => res.ok({}), res.serverError)
+  },
+
+  respond: function (req, res) {
+    var userId = req.session.userId
+    var post = res.locals.post
+    var response = req.param('response')
+
+    EventResponse.query(qb => {
+      qb.where({user_id: userId, post_id: post.id})
+      qb.orderBy('created_at', 'desc')
+      return qb
```

Comment:
you don't need to return qb -- Bookshelf's query method works entirely through side-effects on the argument